### ISSUE TITLE
No card detected after NVIDIA installation via Driver Manager

### ISSUE THREAD

**Issue by fizzgig242** [linuxmint/mint20-beta#178]
After installing and updating Linux Mint 20 I tried to install my integrated NVIDIA card via Driver Manager.
No error messages and after a reboot if I try to start nvidia-settings I get an empty popup window.
I tried to start via terminal but got following error message:

ERROR: NVIDIA driver is not loaded
ERROR: Unable to load info from available system

Tried all different drivers from Driver Manager with the same problem.

inxi -Gxx output: https://pastebin.com/sF5rnNhg

**Comment by clefebvre** [comment]
Did the packages install successfully?

Can you paste the output of `dpkg -l | grep nvidia` and `dpkg -l | grep linux`?

**Comment by fizzgig242** [comment]
dpkg -l | grep nvidia: https://pastebin.com/dWje0bTB

dpkg -l | grep linux: https://pastebin.com/PXQKUcK5

**Comment by clefebvre** [comment]
Everything looks good there..

**Comment by fizzgig242** [comment]
I thought so too: https://imgur.com/pLfrBOh

But I get this: https://imgur.com/aGRC4vr

**Comment by clefebvre** [comment]
What's in `/proc/cmdline`?

**Comment by fizzgig242** [comment]
BOOT_IMAGE=/boot/vmlinuz-5.4.0-37-generic root=UUID=c4b6ae4d-4819-431e-a146-7e34985d13ad ro quiet splash

**Comment by clefebvre** [comment]
Can you connect to IRC in about an hour?

**Comment by fizzgig242** [comment]
Sure! To what channel?

**Comment by clefebvre** [comment]
I'll ask Michael to join us, we both have prime laptops, we'll be able to troubleshoot with you.

#issue178 on Freenode.

**Comment by fizzgig242** [comment]
Hey guys,

after installing Kernel 5.6.19-050619-generic manually, adding the graphics ppa and installing nvidia driver 440 my integrated gpu is working.

Problem now is that the nvidia applet is not installed and offloading is not possible. I think I am missing some packages at the moment.

Any ideas?

**Comment by clefebvre** [comment]
nvidia-prime-applet?

**Comment by clefebvre** [comment]
@fizzgig242 was it 5.6 that made it work or the PPA drivers?

**Comment by fizzgig242** [comment]
> @fizzgig242 was it 5.6 that made it work or the PPA drivers?

I am not sure. Will uninstall nvidia driver, graphics ppa and install nvidia driver from standard repo.

**Comment by fizzgig242** [comment]
OK. After uninstalling ppa nvidia driver, removing ppa and installing driver from repo it works.
Had to install nvidia-prime-applet manually. Don't know why.
It seems it was the Kernel.

**Comment by clefebvre** [comment]
cool.

**Comment by clefebvre** [comment]
on my side I tried the updates (we modified the way mint tools interact with aptdaemon post-beta), I even installed the drivers after applying all updates (including a new kernel) but without rebooting... and it just works.

I think we're looking at a kernel regression with nvidia drivers, but it only affects certain chipsets. I'll close this now because they're not much we can do about it unfortunately.
